### ISSUE TITLE
Loss weights for different datasets

### ISSUE THREAD

**Issue by krishnakanthnakka** [aamir-mustafa/pcl-adversarial-defense#8]
Can you please share the weight-prox,weight-conprox, lr_prox, lr_conprox, lr_model for different datasets.

I find that the experiments are extremely sensitive to prox/conprox related parameters. This will be helpful for reproducing results.

**Comment by aamir-mustafa** [comment]
Hi,

Yes I am planning to share a separate file containing the hyper-parameters chosen while training for all 5 datasets used in the paper.

Thanks for the interest
